src/main/java/com/farao_community/farao/gridcapa/task_manager/TaskNotifier.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.farao_community.farao.gridcapa.task_manager;
+
+import com.farao_community.farao.gridcapa.task_manager.entities.Task;
+import com.farao_community.farao.gridcapa.task_manager.entities.TaskDto;
+import org.springframework.cloud.stream.function.StreamBridge;
+import org.springframework.stereotype.Service;
+
+/**
+ * @author Joris Mancini {@literal <joris.mancini at rte-france.com>}
+ */
+@Service
+public class TaskNotifier {
+
+    private StreamBridge streamBridge;
+
+    public TaskNotifier(StreamBridge streamBridge) {
+        this.streamBridge = streamBridge;
+    }
+
+    public void notifyUpdate(Task task) {
+        streamBridge.send("task-updated", TaskDto.fromEntity(task));
```

Comment:
Create a constant for "task-updated" that shows it is a binding name

Reply:
done